etc/generate-envoy-config/envoy.jsonnet

```diff
@@ -0,0 +1,196 @@
+// Services that we want to proxy; where they are and how to route them.
+local services = {
+  'pachd-grpc': {
+    internal_port: 1650,
+    external_port: 30650,
+    grpc: true,
+    service: 'pachd-proxy-backend',
+    routes: [
+      {
+        match: {
+          grpc: {},
+          prefix: '/',
```

Comment:
so grpc to '/' will go to pachd and '/' without grpc will go to console?

Reply:
Yup.